lib/elixir/lib/task.ex

```diff
@@ -183,18 +183,35 @@ defmodule Task do
     * `:ref` - the task monitor reference
 
     * `:owner` - the PID of the process that started the task
-
```

Comment:
Minor thingy: I would keep this empty line :)

Reply:
@josevalim took care of it